code/src/components/List.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { Task } from "components/Task";
+import styled from "styled-components";
+
+export const List = () => {
+  const taskStore = useSelector(store => store.tasks.tasks);
```

Comment:
This one is a valid time to select the entire list rather than just the length of the array since you're using both further down